biome.jsonc

```
@@ -20,11 +20,12 @@
     "rules": {
       "recommended": true,
       "style": {
-        "noUselessElse": "off", // TODO: turn on, too many cases
-        "noNonNullAssertion": "off" // TODO: turn on, too many cases
+        "noUselessElse": "error",
+        "noNonNullAssertion": "error"
       },
       "suspicious": {
-        "noConsole": "warn"
+        "noConsole": "warn",
+        "useAwait": "error"
```

Sometimes we can make diff smaller like this, so diff are easier to understand in future (like git blame).

```suggestion
        "useAwait": "error",
        "noConsole": "warn"
```